lib/ansible/plugins/callback/default.py

```diff
@@ -214,7 +214,8 @@ def _print_task_banner(self, task):
         if task_name is None:
             task_name = task.get_name().strip()
 
-        self._display.banner(u"%s [%s%s]" % (prefix, task_name, args))
+        checkmsg = " [CHECK MODE]" if task.check_mode or (task.check_mode is None and context.CLIARGS['check']) else ""
```

Comment:
Just for clarity when reading the code, this should probably be broken out into multiple lines. It's a bit too long for a simple ternary (imo).

Reply:
Would this be more appropriate ?
```
if task.check_mode or (task.check_mode is None and context.CLIARGS['check']):
  checkmsg = " [CHECK MODE]"  
else 
  checkmsg = ""
```


I prefer the compact one line form for such a simple assignment statement. Also there are other long lines e.g.
```
if (self._display.verbosity > 2 or '_ansible_verbose_always' in result._result) and '_ansible_verbose_override' not in result._result:
```

Reply:
Yes, that would be preferable.

That other long line you pointed out is *just* if the `if` condition, not the entire if/else construct like the line I pointed out here.

Reply:
Ok, I get yr point.